index.html

```diff
@@ -180,7 +180,38 @@ <h3> <img src="/vectorsImg/Tertiary/Icons/circle.svg" alt=""> Skills</h3>
                 </section>
             </article>
         </section>
+
+        <section class="contact-section">
+            <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+
+            <form action="https://formspree.io/f/mqkngqpl" method="post">
+                <ul>
+                    <li>
+                        <input type="text" id="name" name="user_name" placeholder="Full name" maxlength="30" required>
+                    </li>
+                    <li>
+                        <input type="email" id="mail" name="user_email" placeholder="Email address" required>
+                    </li>
+                    <li>
+                        <textarea name="user_msg" id="msg" maxlength="500" placeholder="Enter text here" required></textarea>
+                    </li>
+                    <li class="btn-container">
+                        <button type="submit">Get in touch</button>
```

Comment:
- [ ] Great job using a button, please add button interaction hover, active (when it's pressed), disabled and enable (default state).

![image](https://user-images.githubusercontent.com/59416136/165383068-a0d75a09-19d9-442b-af8d-502791447845.png)